src/reactionsystem.jl

```diff
@@ -617,70 +628,77 @@ function ReactionSystem(eqs, iv, unknowns, ps;
                         continuous_events = nothing,
                         discrete_events = nothing,
                         metadata = nothing)
-                        
-    name === nothing &&
+               
+    # Error checks
+    if name === nothing
         throw(ArgumentError("The `name` keyword must be provided. Please consider using the `@named` macro"))
+    end
     sysnames = nameof.(systems)
-    (length(unique(sysnames)) == length(sysnames)) ||
-        throw(ArgumentError("System names must be unique."))
+    (length(unique(sysnames)) == length(sysnames)) || throw(ArgumentError("System names must be unique."))
```

Comment:
This kind of change is just going to get undone when we next run the formatter I think. More generally, it is noise in the PR that makes understanding what you've changed harder. Please keep such reformatting type changes to separate PRs to help keep your PRs as short as possible.

Reply:
Agree.

In my defense there was a large number of formatting changes that I wanted to do that I left out for this reason. I only changed these ones because I (apparently erroneously) thought it was an error introduced by the formater.